src/actions/receive-profile.js

```diff
@@ -964,7 +964,7 @@ type FetchProfileArgs = {
   url: string,
   onTemporaryError: TemporaryError => void,
   // Allow tests to capture the reported error, but normally use console.error.
-  reportError?: Function,
+  reportError?: (...data: Array<any>) => void,
```

Comment:
For some reason this generates an error. I believe I tried your way first.

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ src/actions/receive-profile.js:996:62

Cannot call _extractProfileOrZipFromResponse with reportError bound to reportError because mixed [1] is incompatible
with undefined [2] in the return value.

 [1]  967│   reportError?: (...data: Array<any>) => mixed,
         :
      993│     const response = await fetch(url);
      994│     // Case 1: successful answer.
      995│     if (response.ok) {
      996│       return _extractProfileOrZipFromResponse(url, response, reportError);
      997│     }
      998│
      999│     // case 2: unrecoverable error.
         :
 [2] 1059│   reportError: (...data: Array<any>) => void
```